backend/libraries/types/src/cryptocurrency.rs

```diff
@@ -8,27 +8,31 @@ const E8S_PER_TOKEN: u64 = 100_000_000;
 pub enum Cryptocurrency {
     InternetComputer,
     SNS1,
+    CKBTC,
 }
 
 impl Cryptocurrency {
     pub fn token_symbol(&self) -> String {
         match self {
             Cryptocurrency::InternetComputer => "ICP".to_string(),
             Cryptocurrency::SNS1 => "SNS1".to_string(),
+            Cryptocurrency::CKBTC => "ckBTC".to_string(),
         }
     }
 
     pub fn decimals(&self) -> usize {
         match self {
             Cryptocurrency::InternetComputer => 8,
             Cryptocurrency::SNS1 => 8,
+            Cryptocurrency::CKBTC => 8,
         }
     }
 
     pub fn transfer_limit(&self) -> u128 {
         match self {
             Cryptocurrency::InternetComputer => (50 * E8S_PER_TOKEN).into(),
             Cryptocurrency::SNS1 => (10 * E8S_PER_TOKEN).into(),
+            Cryptocurrency::CKBTC => (E8S_PER_TOKEN /100).into(),
```

Comment:
This will fail fmt

Reply:
Damn I must have slipped on the delete key after I'd made the change!